src/pages/tests/Projects.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect'
+
+import { cleanup, render } from '@testing-library/react'
+
+import Projects from '../Projects'
+
+const setup = () => {
+
+    const wrapper = render(
+        <Projects />
+    )
+    return {wrapper}
+}
+
+describe('DeveloperCard', () => {
+
+    afterEach (() => {
+        cleanup()
+    }) 
+
+    it('renders with correct information', async () => {
```

Comment:
Is this test looking for multiple values with `getByAltText` and `getByText` and only checking that `"Soon"` is as expected?

Reply:
Yep. So, once we've only two "different" cards, we need to check if they are rendered as expected and that's why we're using `getByAltText` and `getByText`. Once the other cards are just completing the grid, we just need to know if they're rendered, in the expected quantity (4 times, in this case)

Reply:
What happens to `getByText("BR.TRENDS")` if `"BR.TRENDS"` is not found?

Reply:
Our tests will fail. In this case, the getByText searches for the text we're passing inside the HTML tags, if it doesn't find it throws an error as it on the image below.